pkg/server/server.go

```diff
@@ -607,7 +607,7 @@ func (api *APIServer) httpCreateHandler(w http.ResponseWriter, r *http.Request)
 			api.metrics.LastDuration["create"].Set(float64(time.Since(start).Nanoseconds()))
 			api.metrics.LastFinish["create"].Set(float64(time.Now().Unix()))
 		}()
-		err := backup.CreateBackup(cfg, backupName, tablePattern, schemaOnly, rbacOnly, configsOnly, api.clickhouseBackupVersion)
+		err := backup.CreateBackup(cfg, backupName, tablePattern, "", schemaOnly, rbacOnly, configsOnly, api.clickhouseBackupVersion)
```

Comment:
Please, don't do that

you should add same parameters on API and command Line
please try to figure out how server.go works
and add query string parameter parsing for partitions